content/03.results-and-discussion.md

```diff
@@ -4,41 +4,43 @@
+Secondly, reproducibility is made easier through the use of 'profiles' that can define configuration parameters. These profiles can be managed at different hierarchical levels. HPC-level profiles can specify parameters for the computing environment (job schedulers, cache locations for containers, maximum memory and CPU resources etc.), which can be centrally managed to ensure all users of a group use the same settings. Pipeline-level profiles, specifying default parameters for nf-core/eager itself, allow fast access to routine pipeline-run parameters via a single flag in the nf-core/eager run command, without having to newly configure each new run.
 
-nf-core/eager provides improved efficiency over the original EAGER pipeline by replacing the sample-by-sample sequential processing with Nextflow's asynchronous parallelisation. This, combined with pre-defined per-process customisation of resource parameters, reduces unnecessary resource allocation that can occur with new users to each step of an NGS data processing pipeline. This is particularly pertinent given the increasing use of centralised HPCs or cloud computing, which often use per-hour cost calculations.
+Compared to the original EAGER that utilised per-FASTQ XML files with hardcoded filepaths for a specific user's server, nf-core/eager allows researchers to publish the specific profile used in their runs alongside their publications, to ensure other groups can generate the same results. Usage of profiles also reduces mistakes caused by insufficient 'prose' based reporting of program settings, which may occur in papers written by researchers unfamiliar with informatics. The default nf-core/eager profile uses parameters evaluated in different aDNA specific contexts (e.g. @doi:10.1186/1471-2164-13-178], and will be updated in each new release as new studies are published.
```

Comment:
> , which may occur in papers written by researchers unfamiliar with informatics.

Throwing accusations around ? 😉

Reply:
Honestly, yes. The amount of papers I look for their settings...